src/sentry/models/debugfile.py

```diff
@@ -363,6 +363,23 @@ def _analyze_progard_filename(filename: str) -> Optional[str]:
         return None
 
 
+@region_silo_only_model
+class ProguardArtifactRelease(Model):  # type: ignore
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField(db_index=True)
```

Comment:
Regarding indexing, I would suggest to reason about the size of the result set once an index run is done, that is, if I use the `org_id` in the index, will I get a very small amount of rows to filter sequentially? This depends on data distribution and I have no idea about that, so you might try to think what is the best approach here.

I would expect that having an index on project and org as you did is fine, unless the `proguard_uuid` has mostly unique entries, in that case, it would be better to add an index there.

Of course the best solution would be to use a composite index but it would be useful only in case we always query nearly all composite fields and each field has a very high cardinality, in the other cases it will just occupy more space on disk.

Reply:
In this case proguard uuids are mostly unique. Thx for the insight